app/helpers/application_helper.rb

```diff
@@ -29,4 +29,13 @@ def banner_image_path
     asset_path(pick_a_banner_image)
   end
 
+  def time_since(time)
+    return nil unless time
+
+    formatted_time = time.in_time_zone("Pacific Time (US & Canada)")
+      .strftime("%a, %b %e, %Y %l:%M %p %Z")
+      .gsub("  ", " ")
```

Comment:
You can add `-` before the format directive to tell strftime not to pad the output

```ruby
.strftime("%a, %b %-e, %Y %-l:%M %p %Z") # no need for gsub("  ", " ")
```